web/index.ts

```diff
@@ -276,24 +274,6 @@ createConnection()
             app.use('/api/register', registrationRouter);
         }
 
-        function configureScreenerAPI() {
-            const screenerApiRouter = express.Router();
-            screenerApiRouter.use(screenerAuthCheck);
```

Comment:
do we still need the `screenerAuthCheck`?

Reply:
Good catch, probably not no.